codeSnippets/snippets/client-caching/src/main/kotlin/com/example/Application.kt

```diff
@@ -3,18 +3,23 @@ package com.example
 import io.ktor.client.*
 import io.ktor.client.engine.cio.*
 import io.ktor.client.plugins.cache.*
+import io.ktor.client.plugins.cache.storage.*
 import io.ktor.client.plugins.logging.*
 import io.ktor.client.request.*
 import kotlinx.coroutines.*
+import java.nio.file.*
 
 fun main() {
     runBlocking {
         val client = HttpClient(CIO) {
-            install(HttpCache)
+            install(HttpCache) {
+                val cacheFile = Files.createDirectories(Paths.get("build/cache")).toFile()
+                publicStorage(FileCacheStorage(cacheFile))
```

Comment:
the API got updated a bit, now it's called `FileStorage`

Reply:
thanks, fixed